src/sliding-tab/ExNavigationSlidingTab.js

```diff
@@ -226,6 +228,7 @@ class ExNavigationSlidingTab extends PureComponent<any, Props, State> {
       tabStyle: this.props.tabStyle,
       labelStyle: this.props.labelStyle,
       renderLabel: renderLabelFn,
+      tabBarScrollEnabled: this.props.tabBarScrollEnabled,
```

Comment:
it should be:
 
`scrollEnabled: this.props.tabBarScrollEnabled,`

as you passing it to TabBar